pkg/virt-api/rest/profiler.go

```diff
@@ -135,6 +135,8 @@ func podIsReadyComponent(pod *k8sv1.Pod) bool {
 		return false
 	} else if pod.Status.Phase != k8sv1.PodRunning {
 		return false
+	} else if pod.DeletionTimestamp != nil {
```

Comment:
I didn't look too close, but I suspect that  the test failed everytime if a pod got deleted and then it failed to determine whether the pod was ready somehow?